platform/backend/api/src/integrationTest/kotlin/io/hamal/api/http/extension/ExtensionCreateControllerTest.kt

```diff
@@ -0,0 +1,31 @@
+package io.hamal.api.http.extension
+
+import io.hamal.lib.domain.vo.CodeId
+import io.hamal.lib.domain.vo.CodeVersion
+import io.hamal.lib.domain.vo.ExtensionId
+import io.hamal.lib.domain.vo.ExtensionName
+import io.hamal.lib.sdk.api.ApiCreateExtensionReq
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import org.junit.jupiter.api.Test
+
+internal class ExtensionCreateControllerTest : ExtensionBaseControllerTest() {
+    @Test
+    fun `Creates extension`() {
+        val res = awaitCompleted(
+            createExtension(
+                ApiCreateExtensionReq(
+                    name = ExtensionName("TestExtension"),
+                    codeId = CodeId(1),
+                    codeVersion = CodeVersion(1)
```

Comment:
see ApiCreateFuncReq -> code value instead
```suggestion
code = CodeValue("x='hamal'")
```